tests/dummy/app/controllers/application.js

```diff
@@ -0,0 +1,42 @@
+import Ember from 'ember';
+
+let reg = /^%\[([^\]]*)\]\s*\(([^)]+)\)/;
+
+export default Ember.Controller.extend({
+  mdPlugins: [
```

Comment:
If you'd like, you can also use this in [dummy app application.hbs](https://github.com/johnotander/ember-remarkable/blob/master/tests/dummy/app/templates/application.hbs) to verify that it's working (dummy app is served with `$ ember serve` for addons).